css/style.css

```diff
@@ -204,10 +333,10 @@ body {
 .nav ul li {
 	display: inline-block;
 	margin: 0px 5px 0px 5px;
-}
-
-.nav ul li a {
-	text-decoration: none;
```

Comment:
I thought that the admin navigation links needed to have the underline, the added css below is for the fonts used in the admin navigation